melodic/distribution.yaml

```diff
@@ -8280,7 +8280,7 @@ repositories:
       tags:
         release: release/melodic/{package}/{version}
       url: https://github.com/SICKAG/sick_scan-release.git
-      version: 1.3.21-0
+      version: 1.3.21-2
```

Comment:
It's unusual to just bump the debinc, especially since it looks like there have been a lot of changes since the last release in May. Did you forget to [bump the version number](https://github.com/SICKAG/sick_scan/blob/eb686072668ca407d6f96f095ce38e6e994e1b91/package.xml#L4) and [update the changelog](https://github.com/SICKAG/sick_scan/blob/master/CHANGELOG.rst)?